lib/v2/gamegene/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'gamegene.cn/': {
+        _name: '游戏基因 GameGene',
+        news: [
+            {
+                title: '资讯',
+                docs: 'https://docs.rsshub.app/routes/game#gamegene-zi-xun',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/f045902f05323a65f612ddb650f6946473943eb5/website/docs/routes/game.md?plain=1#L1010

```suggestion
                docs: 'https://docs.rsshub.app/routes/game#you-xi-ji-yin',
```